oauth2client/contrib/appengine.py

```diff
@@ -477,15 +469,15 @@ def _parse_state_value(state, user):
         user: google.appengine.api.users.User, The current user.
 
     Raises:
-        InvalidXsrfTokenError: if the XSRF token is invalid.
+        ValueError: if the XSRF token is invalid.
```

Comment:
> The smell I'm getting is that it should be either privatized or dropped. @jonparrott, am I making sense?

Right.

I'd prefer if the xsrf check function didn't raise an exception but rather returned True of False. Exceptions are for exceptional circumstances.